pkg/defaults/defaults.go

```diff
@@ -23,6 +23,8 @@ var (
 	K0sBinaryURL = ""
 )
 
+const KOTSADM_NAMESPACE = "kotsadm"
```

Comment:
As per convention shouldn't this be named KotsadmNamespace or something like it ?

Reply:
Globals are named like this everywhere in my experience?

Reply:
will do